website/static/js/tests/nodeControl.test.js

```diff
@@ -0,0 +1,83 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+var assert = require('chai').assert;
+var assign = require('object-assign');
```

Comment:
In browser tests, we can save an import (and thus increase build speed) by using `$.extend` rather than `object-assign`.